test_settings.py

```diff
@@ -88,6 +88,8 @@ def root(*args):
 
 LMS_ROOT_URL = "http://localhost:8000"
 
+OAUTH_ID_TOKEN_EXPIRATION = 60*60
```

Comment:
@pomegranited nit: might make sense to include measurement unit into the var name - to make it clear it's in seconds.